.gitignore

```diff
@@ -23,62 +24,24 @@
 
 # Flutter/Dart/Pub related
 **/doc/api/
+**/ios/Flutter/.last_build_id
 .dart_tool/
 .flutter-plugins
 .flutter-plugins-dependencies
-.packages
 .pub-cache/
 .pub/
 /build/
 
-# Android related
-**/android/**/gradle-wrapper.jar
-**/android/.gradle
-**/android/captures/
-**/android/gradlew
-**/android/gradlew.bat
-**/android/local.properties
-**/android/**/GeneratedPluginRegistrant.java
-
-# iOS/XCode related
```

Comment:
Why touch this? I would definitely like to preserve iOS ignores

But other changes to this file are questionable as well

UPD: apparently it's because of the generated `android/.gitignore`. I would rather have a unified .gitignore file

Reply:
My workflow when updating the project is like this:
* I create a new empty Flutter project.
* I do a directory comparison between our project and the newly generated project.
* I manually go through the diff and check what should be changed.

The new flutter project template splits the unified `.gitignore` and moves the Android specific ignores into `android/.gitignore` and the IOS related ones into `ios/.gitignore`. I'm trying to bring our project as close as possible to the Flutter project template, because it reduces the noise of our own changes in the diff and therefore makes it easier for me to spot the actual important changes. And if we don't align with these changes, they'll show up in the diff of every future update.

On a side note, I am personally in favor of moving things to a closer .gitignore if possible, since it reduces the chance that a rename of a parent directory will break something.